nam/train/gui.py

```diff
@@ -112,6 +115,7 @@ def _set_val(self):
         res = {
             _PathType.FILE: filedialog.askopenfilename,
             _PathType.DIRECTORY: filedialog.askdirectory,
+            _PathType.MULTIFILE: filedialog.askopenfilenames,
```

Comment:
Nit: I'd love if the tuple were stripped in the displayed text if only one output file were selected.

Reply:
Done - still displays tuple format for multiple files